packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -221,6 +221,44 @@ class VmServiceWrapper implements VmService {
     }
   }
 
+  //{ "_setTraceClassAllocation", SetTraceClassAllocation,
+  Future<Success> setTraceClassAllocation(
+    String isolateId, {
+    String classId,
+    bool enable,
+  }) async {
+    final Map<String, dynamic> args = {};
+    if (classId != null) {
+      args['classId'] = classId;
+    }
+    if (enable != null) {
+      args['enable'] = enable ? true : false;
+    }
+
+    final response = await trackFuture(
```

Comment:
Fyi @bkonyi we need to make these or similar APIs public instead of using a private API.

Reply:
Yeah, I'd strongly prefer that we avoid using private methods outside of VM developer mode. I'm working on a public implementation now, so it would be great if we can wait on landing this PR until we have that in place. I'm much more comfortable having a DevTools out in the wild that checks to see if there's support for `setTraceClassAllocation` and falls back to `_setTraceClassAllocation` than just blindly calling `_setTraceClassAllocation`, which will likely be renamed and break with newer service targets.

Reply:
okay, let me know when its ready.